core-java-modules/core-java-arrays-sorting/src/test/java/com/baeldung/sorted/operations/InsertElementInSortedArrayTest.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.sorted.operations;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class InsertElementInSortedArrayTest {
+
+    private static final int[] SORTED_ARRAY = new int[] {4,55,85,100,125};
+    
+    @Test
+    public void givenSortedArray_whenElementInserted_thenArrayRemainsSorted(){
```

Comment:
You have a number of formatting issues - please ensure to apply the Baeldung auto-formatter
```suggestion
    public void givenSortedArray_whenElementInserted_thenArrayRemainsSorted() {
```